yt_dlp/options.py

```diff
@@ -1378,8 +1378,8 @@ def _alias_callback(option, opt_str, value, parser, opts, nargs):
         help='Sanitize filenames only minimally')
     filesystem.add_option(
         '--trim-filenames', '--trim-file-names', metavar='LENGTH',
-        dest='trim_file_name', default=0, type=int,
-        help='Limit the filename length (excluding extension) to the specified number of characters')
+        dest='trim_file_name', default='none',
+        help='Limit the filename length (excluding extension) to the specified number of characters or bytes')
```

Comment:
Maybe you are missing explaining the way to go from character mode to bytes. I'll propuse this:

```suggestion
        help='Limit the filename length (excluding extension) to the specified number of characters or bytes (end the number with b for bytes)')
```